build-config.yaml

```diff
@@ -6,4 +6,4 @@ required-versions:
   # NB. The Rust version is pinned in rust-toolchain.toml.
   # NB. The Node version is pinned in .node-version.
   cargo-watch: ^8.1.1
-  wasm-pack: ^0.12.1
+  wasm-pack: ">=0.12.1"
```

Comment:
Why this change? Are not afraid of wasm-pack updates even a bit?

Reply:
I am using 0.13 and there is no issue with that. Historically they also didn't really make much updates. 0.13 was released a few weeks ago and it was a first release since a few years back. For CI, we pin the installed version anyway. This change just makes the potential compatibility issues with local setup go away. We are phasing out wasm-pack usage completely soon enough anyway.